app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,14 @@
 <resources>
     <string name="app_name">WNews</string>
+    <string name="title_activity_login">LoginActivity</string>
```

Comment:
Strings in this file are meant to be easily translated. That's why the go on a different file, so they can be forwarded to a translator so he/she may translate them if needed.

For this reason, it helps a lot that strings have more context to where they are used in the app.

This file could be improved by grouping strings like this:

```xml

<!-- Login -->
<string name="login_user_email">Email</string>
<string name="login_user_password">Password</string>
...


<!-- Signup -->
<string name="signup_welcome_message">Bienvenido a Wolox!</string>
<string name="signup_error_invalid_email">A valid email is required</string>
....
```

Please notice not only how I grouped strings by their context in the app, but also how I added prefixes and tried to put the thing that some strings have in common before in the ID. Like `loing_user_password` and `login_user_email`. 

Doing all of this helps the team maintain a consistent style through the app's development cycle, reduces endless debates about naming conventions and makes it easier for newcomers to join the project, as they can catch up of how the app should be coded faster.